index.html

```diff
@@ -135,6 +135,32 @@ <h4>EditContext state</h4>
               from the [=Text Input Service=]. The user agent will indicate which positions are
               required by firing {{CharacterBoundsUpdateEvent}}.
             </p>
+            <p>
+              [=Control bounds=], [=selection bounds=], and [=codepoint rects=] are given in the
+              <dfn>client coordinate system</dfn>, which is defined as a two-dimensional Cartesian
```

Comment:
> client coordinate system

is this in CSS pixels? https://drafts.csswg.org/css-values-4/#px

Reply:
Added link to CSS pixel definition.